pyproject.toml

```diff
@@ -30,21 +30,23 @@ classifiers=[
 urls = {Homepage = "https://github.com/Alice-Bob-SW/qiskit-alice-bob-provider", "Alice & Bob" = "https://alice-bob.com/"}
 requires-python = ">=3.8, <3.12"
 dependencies = [
+    "ipython",
```

Comment:
We must not set a fixed version of ipython, to avoid conflicts with users installing the library that already have their own version.

More importantly, if you install the provider from a running notebook (including on colab), it would fail as it will tell you the ipykernel module can not be changed while running. This can go as far as crashing Colab haha.